### PR TITLE
WIP - Add Mapillary demo

### PR DESCRIPTION
**Pull request by tobrun**
Example using Mapillary data

### CODE REVIEW

File: MapboxAndroidDemo/src/gpservices/AndroidManifest.xml [hunk 1]
```diff
@@ -401,5 +401,14 @@
             </intent-filter>
         </activity>
 
+        <activity
+            android:name=".labs.MapillaryActvitiy"
+            android:label="@string/activity_camera_restrict_title"
```
Comment (langsmith) [inline MapboxAndroidDemo/src/gpservices/AndroidManifest.xml]: Needs to be `activity_labs_mapillary_title`

**Comment by tobrun** [comment]
ezgif com-video-to-gif 77

**Comment by langsmith** [comment]
😍

File: MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java [hunk 2]
```diff
@@ -15,7 +15,7 @@
 import com.mapbox.mapboxandroiddemo.examples.OfflineMapActivity;
 import com.mapbox.mapboxandroiddemo.examples.SimpleMapViewActivity;
 import com.mapbox.mapboxandroiddemo.model.ExampleItemModel;
-import com.mapbox.mapboxandroiddemo.utils.OffsettingHelper;
+import com.maPpbox.mapboxandroiddemo.utils.OffsettingHelper;
```
Comment (zugaldia) [inline MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java]: TyPpo.

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java [hunk 3]
```diff
@@ -539,6 +540,12 @@ private void listItems(int id) {
       case R.id.nav_lab:
         exampleItemModel.add(null);
         exampleItemModel.add(new ExampleItemModel(
+          R.string.activity_labs_mapillary_title,
+          R.string.activity_labs_mapillary_title,
```
Comment (langsmith) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java]: Should be `activity_labs_mapillary_description`

Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java]: Fixed

File: MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java (hunk 2 above)
Comment (langsmith) [reply MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java]: Fixed

**Comment by langsmith** [comment]
Ready to go @zugaldia .

**Pull request merged**